### ISSUE TITLE
Error when using hup-ckmsg

### ISSUE THREAD

**Issue by mcblum** [redoPop/husky-pivotal#1]
Thanks for writing these! Have you ever seen this error?

```
**  return process.env.GIT_PARAMS.split(' ')[index];
                                ^

TypeError: Cannot read property 'split' of undefined
**
```

Error is coming from `husky-pivotal/lib/git.js:26:33`

**Comment by redoPop** [comment]
Ooh: [looks like I need to update husky-pivotal for changes in Husky's 1.x release candidates.](https://github.com/typicode/husky/blob/c900d557c3ec7aaf508dd12219da95c7da8b0dd2/CHANGELOG.md#100-rc1)

**Comment by redoPop** [comment]
I just published [Husky Pivotal 1.0.1](https://www.npmjs.com/package/husky-pivotal/v/1.0.1) which includes changes from #3 that I suspect will address this issue:

My suspicion is that you're using a Husky 1.x release candidate, where the `GIT_PARAMS` node environment variable has been renamed to `HUSKY_GIT_PARAMS` to avoid collision.

As a bonus, I've also extended peer dependencies coverage in 1.0.1 so you shouldn't get warnings for using an unexpected Husky version anymore. 😊

Please let me know if husky-pivotal v1.0.1 doesn't address your issue – it may be that there's something else different about your environment.
